### PR TITLE
Fix serializer required fields and validations for empty values

### PR DESCRIPTION
**Pull request by KhaledBousrih**

### CODE REVIEW

File: concrete_datastore/api/v1/validators.py [hunk 1]
```diff
@@ -1,7 +1,58 @@
...
 from django.core.exceptions import ValidationError
 
 
 def validate_file(fieldfile_obj):
     if fieldfile_obj.file is None:  # skip-test-coverage
-        raise ValidationError("Erreur : fichier absent.")
+        raise ValidationError("Error : missing file.")
+
+
+def validate_required(field):
```
Comment (JulienTanneau) [inline concrete_datastore/api/v1/validators.py]: Naming must be explicit. In this case you return a function. So name it like `get_validate_field_function`

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 ### Changed
 
-- nothing changed
+- changed serializer fields validations
```
Comment (lcognat) [inline CHANGELOG.md]: Description of changes is not specific enough

File: concrete_datastore/api/v1/validators.py (hunk 1 above)
Comment (lcognat) [reply concrete_datastore/api/v1/validators.py]: ou plutôt `get_field_validator`

File: concrete_datastore/concrete/constants.py [hunk 3]
```diff
@@ -63,3 +64,14 @@
...
+EMPTY_VALUES_MAP = {
+    'TextField': {'empty_value': '', 'field_type': str},
+    'CharField': {'empty_value': '', 'field_type': str},
+    'IntegerField': {'empty_value': 0, 'field_type': int},
+    'FloatField': {'empty_value': 0.0, 'field_type': float},
+    'DecimalField': {'empty_value': Decimal(0), 'field_type': Decimal},
+    'JSONField': {'empty_value': {}, 'field_type': dict},
+    'BigIntegerField': {'empty_value': 0, 'field_type': int},
+    'BooleanField': {'empty_value': None, 'field_type': bool},
+}
```
Comment (JulienTanneau) [inline concrete_datastore/concrete/constants.py]: Remove all and keep only TextField and Charfield. cf validators.py

File: CHANGELOG.md (hunk 2 above)
Comment (KhaledBousrih) [reply CHANGELOG.md]: added more details

File: concrete_datastore/concrete/constants.py (hunk 3 above)
Comment (KhaledBousrih) [reply concrete_datastore/concrete/constants.py]: removed. The only keys kept are `CharField` `TextField`.

**Pull request merged**